[foo.assertion.1] line 8 assertion Integer (Five) > 0: SUCCESS
VERIFICATION SUCCESSFUL
